Blog post with second cohort projects (#1003)
ropensci/roweb3: content/blog/2025-04-15-champions-project-wrapup/index.md

params:
  doi: "10.59350/268cb-p0021"
---

Our second cohort of the [rOpenSci Champions Program](/champions/) has now completed the second phase of the program by developing their project and carrying out outreach activities. In this article, we share each Champion's project, their achievements and their outreach activities.

## Champions' Projects

In their applications Champions choose to develop a new package or to participate in the review process as authors or reviewers.

### New packages 

- **Andrea Gomez Vargas**, from Argentina, developed the package [arcenso](). to make Argentinian historical census data available, homogenized, and ready to use.  Her mentor is [Luis Verde]() from Mexico. 

- **Francesca Belem Lopes Palmeira**, from Brazil, plans to develop a new package called 'Potential for Conflict Index - PCI', which is a descriptive statistical method used to enhance the understanding of outcomes in human dimensions research. The package will equip users with the tools required for calculating, comparing, and graphing the potential for conflict. Her mentor is Bruna Wundervald from Brazil. 

> - **Sunny Tseng**, from Taiwan, developed [bbsTaiwan](), a package to streamline Taiwan BBS data retrieval and analysis. 'bbsTrim' will support data retrieval from GBIF, where Taiwan BBS data is stored. 'bbsTrim' will also include population trend modeling, using TRIM (Trends and Indices for Monitoring data). Users can tailor species analysis, controlling data stratification, preparation, and modeling.

Author: ```suggestion
- **Sunny Tseng**, from Taiwan, developed [bbsTaiwan](https://sunnytseng.github.io/bbsTaiwan/), a package to streamline Taiwan BBS data retrieval and analysis. 'bbsTaiwan' support data retrieval from GBIF, where Taiwan BBS data is stored. The package also include population trend modeling, using TRIM (Trends and Indices for Monitoring data) and mapping. Users can tailor species analysis, controlling data stratification, preparation, and modeling. Her mentor was [Eunseop Kim.](/author/eunseop-kim/)
```

---
title: "rOpenSci Champions Second Cohort: Projects Wrap-Up"
slug: "champions-program-projects-cohort2"
author:
  - Yanina Bellini Saibene
editor:

> - Steffi LaZerte

Author: ```suggestion
- Maëlle Salmon
- Mark Padgham
```

date: '2025-07-02'
tags:
  - community
  - champions program
  - packages

> description: "Our second cohort of Champions complete the program. In this blog post, we share each champions projects, their achievements and outreach activities."

Reviewer: ```suggestion
description: "Our second cohort of Champions completed the program. In this blog post, we share each champion's projects, their achievements and outreach activities."
```

params:
  doi: "10.59350/268cb-p0021"
---

As we make the presentation of our third cohort of the [rOpenSci Champions Program](/champions/),our second cohort has now completed the second phase of the program by developing their project and carrying out outreach activities. In this article, we share each Champion's project, their achievements and their outreach activities.

## Champions' Projects

In their applications Champions choose to develop a new package or to participate in the review process as authors or reviewers.

{{< figure src = "proyectos.png" alt = "Logos of all the packages developed or improved during the champions program 2023-2024" >}}

### New packages 

> - **Andrea Gomez Vargas**, from Argentina, developed the package [arcenso](https://soyandrea.github.io/arcenso/). to allow access to the official data of the national population censuses in Argentina from the National Institute of Statistics and Census - INDEC and make this historical census data available, homogenized, and ready to use.  Her mentor is [Luis Verde](/author/luis-verde-arregoitia/) from Mexico. You can learn more about the package in [the blog post Andrea wrote about her project and her journey on the program](/blog/2025/05/15/puentes-comunidades-campeones-ropensci/).

Reviewer: ```suggestion
- **Andrea Gomez Vargas**, from Argentina, developed the package [arcenso](https://soyandrea.github.io/arcenso/), to provide access to the official data of the national population censuses in Argentina from the National Institute of Statistics and Census - INDEC, and to make this historical census data available, homogenized, and ready to use.  Her mentor was [Luis Verde](/author/luis-verde-arregoitia/) from Mexico. You can learn more about the package in [the blog post Andrea wrote about her project and her journey on the program](/blog/2025/05/15/puentes-comunidades-campeones-ropensci/).
```

> - **Francesca Belem Lopes Palmeira**, from Brazil, developed a new package called [pcir: 'Potential for Conflict Index - PCI'](https://fblpalmeira.github.io/pcir/), which is a descriptive statistical method used to enhance the understanding of outcomes in human dimensions research. The package equip users with the tools required for calculating, comparing, and graphing the potential for conflict. Her mentor is [Bruna Wundervald](/author/bruna-wundervald/) from Brazil. 

Reviewer: "mentor was" - right? It's about the program having been completed, so should be past tense. Or are the mentors still involved? And if so, maybe there needs to be a sentence at the start saying that's phase 2 is not the end, and they will still <... do whatever ...>

- **Sunny Tseng**, from Taiwan, now living in Canada, developed [bbsTaiwan](https://sunnytseng.github.io/bbsTaiwan/). This is an R package to streamline Taiwan Breeding Bird Survey (BBS) data retrieval and analysis. bbsTaiwan facilitates data access from GBIF, where Taiwan BBS data are stored. Users can tailor species analysis, controlling data stratification, preparation, and mapping. Sunny's mentor is [Eunseop Kim](/author/eunseop-kim/) from South Korea, now living in the USA.

> - **Erika Siregar**, from Indonesia, worked on developing [rplaywright](https://erikaris.github.io/rplaywright/) an R package seamlessly bridge with the NodeJS library, Playwright, enabling simplified web testing and automation for R users. This package aims to provide R users with a comprehensive toolkit for interacting with web browsers programmatically, allowing for tasks such as browser automation, web scraping, and end-to-end testing directly from within the R environment. Her mentor is [Apoor Anand](/author/apoorv-anand/) from India. 

Author: I will ask her if she finally present something. Because she was having VISA issues.

### Improving and reviewing existing packages

- **Ezekiel Adebayo Ogundepo** from Nigeria, worked on the [bulkreadr](https://docs.ropensci.org/naijR/) package, which is a package designed to optimize the process of importing and processing large datasets provinding compatibility with several formats files from several statistical softwares and spreadsheets. Ezekiel worked on enhanced functions functionality and lunch the [official website of the package](https://gbganalyst.github.io/bulkreadr/news/index.html). His mentor was [Francisco Cardozo](/author/francisco-cardozo/) from Colombia.

- **Liz Hare** from the US, worked as reviewer on the software peer review of the [emodnet.wfs](https://github.com/ropensci/software-review/issues/653) package, provide helpers to access EMODnet Web Feature Service data through R.  Her mentor was [François Michonneau](/author/françois-michonneau/) from France.

> - **Sehrish Kanwal**, from Australia, worked to improve the [RNAsum R package](https://github.com/umccr/RNAsum), an RNA-seq reporting workflow designed to post-process, summarise, and visualize outputs. Sehrish worked on the package and a paper about it. [The package website](https://umccr.github.io/RNAsum/) present more information on the packages functionalities. Her mentor was [Emi Tanaka](/author/emi-tanaka/) from Australia. 

Reviewer: "website present more information" -> "website presents more information"
"packages functionalities" -> "package's functionality"

- **Binod Jung Bogati**, from Nepal, worked on the [nepalcensus](https://github.com/rugnepal/nepalcensus) package, which provides a comprehensive and up-to-date tool for accessing and exploring demographic, socio-economic, and geographic data collected during the most recent census conducted in Nepal. The package aims to facilitate data analysis and visualization for researchers, policymakers, and anyone interested in understanding and utilizing census data effectively for various applications and insights. His mentor is [Erle Holgersen](/author/erle-holgersen/) from Canada.

## Outreach Activities

{{< figure src = "outreach.png" alt = "Examples of outreach activities: workshops, webinars, conferences, meetups, papers, talks and user groups" >}}

> Our Champions carried out a wide range of outreach activities like talks, conference papers, workshops, blog articles, community calls and meetups. The objective of these activities was to share what they did and learned during the program with their communities beyond rOpenSci.  

Reviewer: ```suggestion
Our Champions carried out a wide range of outreach activities including talks, conference papers, workshops, blog articles, community calls and meetups. The objective of these activities was to share what they did and what they learned during the program with their communities beyond rOpenSci.  
```

- Ezekiel participated as a mentor in the [Coworking Mini-Hackathon for First-Time Contributors](https://ropensci.org/blog/2025/01/24/coworking-hackathons/), helping people to do their first contribution to R packages. 
- Francesca was the main host of our first community call in Portugues [The R community speaks Portuguese](https://ropensci.org/commcalls/translation-portuguese/) and also one of the mentors of our [Tradução + Hackathon = Traslaton rOpenSci](https://www.eventbrite.cl/e/traducao-hackathon-traslaton-ropensci-tickets-1045507530167) during LatinR 2024.

Champions also wrote blog posts for rOpenSci's website covering technological aspects such as [“Resources For Using R With Screen Readers”](https://ropensci.org/blog/2024/09/05/screen-readers-tools/) by Liz Hare and also aspects such as the experience of being part of the program, such as [“Bridges and Communities. My Journey as an rOpenSci Champion”](https://ropensci.org/blog/2025/05/15/puentes-comunidades-campeones-ropensci/) by Andrea Vargas Gomez.

> Academic publication are also part of the outcomes of the Champions work. For example, Sehrish Kanwal wrote a paper about the [RNAsum R package](https://doi.org/10.1101/2025.01.10.24319650).

Reviewer: "wrote a paper" -> "published a paper" ?

> LatinR, useR!, Posit and several other conferences were attended by our champions. For example, during LatinR, Government Advances in Statistical Programming Andrea presented the package she developed during the program. 

Reviewer: ```suggestion
LatinR, useR!, Posit and several other conferences were attended by our champions. For example, Andrea presented the package she developed during the program at LatinR, in the Government Advances in Statistical Programming section. 
```
I didn't understand this, and have guessed it was a section of that conference, so you might need to correct that?

In addition Sunny, Andrea and Erika received scholarships to attend Posit Conf 2024 and Sunny and Andrea were able to participate.
Erika and Andrea also presented their respective packages developed during the program at useR! 2025.

## Champions' accomplishments and next steps

Our Champions have been very successful in the delivery of their projects, building high quality software with a focus on their countries and discipline. The impact of their work also spread to their home institutions and related local communities. Not only through their outreach activities sharing information and knowledge contextualized for their audience, but also by creating spaces for their local communities, like the R user groups, and through their research.

> Many of our Champions will continue with their outreach activities and maintaining their R package. Stay tuned to our events and talks section in our website, our newsletter and social media to learn about them! 

Reviewer: ```suggestion
Many of our Champions will continue with their outreach activities and maintenance of their R package. Stay updated about their activities through our events and talks section in our website, our newsletter and social media! 
```

## Outreach Activities

{{< figure src = "outreach.png" alt = "Examples of outreach activities: workshops, webinars, conferences, meetups, papers, talks and user groups" >}}

Our Champions carried out a wide range of outreach activities including talks, conference papers, workshops, blog articles, community calls and meetups. The objective of these activities was to share what they did and what they learned during the program with their communities beyond rOpenSci.  

Outreach activities commenced in August 2024, involved local communities (universities, RUGs, R-Ladies Chapters, conferences, and organizations), and were delivered in the Champions’ native/preferred languages (English, Spanish, Portuguese and Chinese).

[Liz Hare, member of this cohort worked together with Cagri, a champion from the first cohort to do outreach activities](https://ropensci.org/events/rsr-learn-and-use/) and also introduce their cohort mates to other communities and organizations giving them [space to share what they did during the Champions Program](https://www.youtube.com/watch?v=p2Odyp9VIPA).

> Victor Ordu, first cohort champion, participated in the Unilorin meetup organized by Ezekiel, speaking about [Hands-On Data Cleaning in R with Regular Expressions](https://youtu.be/N0DdbszMt2Y?si=3eZd-G1z-AcYW_TQ(

Author: ```suggestion
[Victor Ordu](/author/victor-ordu/), first cohort champion, participated in the Unilorin meetup organized by Ezekiel, speaking about [Hands-On Data Cleaning in R with Regular Expressions](https://youtu.be/N0DdbszMt2Y?si=3eZd-G1z-AcYW_TQ)
```
